static/src/javascripts/projects/commercial/modules/third-party-tags/outbrain.spec.js

```diff
@@ -0,0 +1,221 @@
+// @flow
+import config from 'lib/config';
+import checkMediator from 'projects/common/modules/check-mediator';
+import { init } from './outbrain';
+import { getSection } from './outbrain-sections';
+
+jest.mock('ophan/ng', () => ({ record: () => undefined }));
+jest.mock('lib/detect', () => ({
+    adblockInUse: Promise.resolve(false),
+    getBreakpoint: jest.fn(),
+}));
+jest.mock('lib/load-script', () => ({ loadScript: jest.fn() }));
+jest.mock('./outbrain-load', () => ({ load: jest.fn() }));
+jest.mock('./outbrain-tracking', () => ({ tracking: jest.fn() }));
+
+const detect = require('lib/detect');
```

Comment:
Ah I remember now - if you put imports after the mocks the linter complains that they aren't at the top of the file